kuma/wiki/views/edit.py

```diff
@@ -167,11 +162,11 @@ def edit(request, document_slug, document_locale, revision_id=None):
                 if doc_form.is_valid():
                     # if must be here for section edits
                     if 'slug' in post_data:
-                        post_data['slug'] = join_slug(
-                            slug_dict['parent_split'], post_data['slug'])
+                        post_data['slug'] = u'/'.join([slug_dict['parent'],
+                                                       post_data['slug']])
 
                     # Get the possibly new slug for the imminent redirection:
-                    doc = doc_form.save(None)
+                    doc = doc_form.save(parent=None)
```

Comment:
:smile: This is why Python has named parameters - make this sort of thing easier to understand.